src/analysis/special_functions/integrals.lean

```diff
@@ -400,6 +400,60 @@ end
 lemma integral_one_div_one_add_sq : ∫ x : ℝ in a..b, 1 / (1 + x^2) = arctan b - arctan a :=
 by simp only [one_div, integral_inv_one_add_sq]
 
+section rpow_cpow
+open complex
+
+lemma integral_mul_cpow_one_add_sq {t : ℂ} (ht : t ≠ -1) :
+  ∫ x : ℝ in a..b, (x:ℂ) * (1 + x ^ 2) ^ t =
+  (1 + b ^ 2) ^ (t + 1) / (t + 1) / 2 - (1 + a ^ 2) ^ (t + 1) / (t + 1) / 2 :=
```

Comment:
The spelling with `/ (t + 1) / 2` looks a little bit strange. Why not `/ (2 * (t + 1))`?

Reply:
I wasn't aware that one was preferred over the other. But the second version slightly shortens the proof so I'm happy to go with it.